src/Components/Alert/Hooks/__tests__/AlertProvider.jest.tsx

```diff
@@ -88,26 +79,39 @@ describe("AlertProvider", () => {
       Viewer: () => ({
         previewSavedSearch: {
           labels: [
-            {
-              displayValue: "Andy Warhol",
-              field: "artistIDs",
-              value: "artist-id",
-            },
+            { displayValue: "Andy Warhol" },
+            { displayValue: "Limited edition" },
           ],
         },
       }),
     }
 
+    const mockedSuggestionResolver = {
+      PreviewSavedSearch: () => ({
+        suggestedFilters: [
+          { displayValue: "Photography" },
+          { displayValue: "Print" },
+        ],
+      }),
+    }
```

Comment:
George helped me solve this by correcting the nesting here to mock [the operation in question](https://github.com/artsy/force/blob/bc9b7d2e1c3304bf4abfad34522fea0f7185e36d/src/Components/Alert/Components/Form/SuggestedFilters.tsx#L121-L134)

The above is correct, whereas the following was not:
```tsx
const mockedSuggestionResolver = {
      Viewer: () => ({.  // ❌
        previewSavedSearch: {
          suggestedFilters: [
            { displayValue: "Photography" },
            { displayValue: "Print" },
          ],
        }
      }),
    }
```

cc @damassi